sites/friday/src/get-started/here/make-changes.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Make changes to your project
+sidebarTitle: Revisions
+weight: 50
+description: How to work on a daily basis with an {{% vendor/name %}} project?
+---
+
+Now that your project is deployed, you can start making changes to it.
+For example, you might want to fix a bug or add a new feature.
+It's this component of Upsun - preview environments - that you'll come to recognize as a key feature.
+
+In your project, the `main` branch (or `master` if you change it during project creation) always represents the production environment.
+Other branches are for developing new features, fixing bugs, or updating the infrastructure.
+
+To make changes to your project, follow these steps:
+
+1. Create a new environment (a Git branch) to make changes without impacting production:
+{{< codetabs >}}
++++
+title={{% vendor/name %}} Git repository
++++
+   ```bash {location="Terminal"}
+   {{% vendor/cli %}} branch feat-a
+   ```
+   This command creates a new local `feat-a` Git branch based on the `main` Git branch
+   and activates a related environment on {{< vendor/name >}}.
+   The new environment inherits the data (service data and assets) of its parent environment (the production environment here).
+
+   {{< note title="Warnings" >}}
```

Comment:
If we change to having the user use git for branching instead of our cli, then we can move the warning out of this tab, and below it's conclusion since the warning is applicable to both scenarios.

Reply:
no, unfortunately i think it's even *more* confusing now. 

My suggestion is to remove the codetabs for `Upsun Git repository` and `Using third party provider`, remove `upsun branch feat-a` and replace it with `git branch checkout -b feat-a`.

This way, it doesn't matter if they're using the Upsun repository or a 3rd party, the command is the same. And given at this stage nothing has changed, there really isn't a need to push yet.

We can move all of this down to step 4 when it's time to push. At that point, we can split between the two scenarios.